packages/react/src/components/input/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {InputHTMLAttributes, LabelHTMLAttributes} from 'react'
+
+interface InputOptions {}
+
+interface InputProps
+  extends InputHTMLAttributes<HTMLInputElement>,
+    InputOptions {
+  name: string
+  label: string
+}
+
+const Label: React.FC<LabelHTMLAttributes<HTMLLabelElement>> = ({
+  children,
+  ...rest
+}) => {
+  return (
+    <label data-sr-input-label {...rest}>
+      {children}
+    </label>
+  )
+}
+
+const Input: React.FC<InputProps> = ({name, label, ...rest}) => {
```

Comment:
This could be more complex but I figured it's good enough for our current use cases — abstracting input label and visual indication of required field.